functional/SyntheticGCWorkload/src/net/adoptopenjdk/casa/workload_sessions/Workloads.java

```diff
@@ -370,7 +375,7 @@ public String toString()
 	/**
 	 * A thread that displays status updates to the user. 
 	 *
-	 *  
+	 * @author Andrew Somerville <andrew.somerville@unb.ca>
```

Comment:
Please remove author tags in source code, we have kept Andrew's name in the docs, but as we open-source code, we strip out author tags in src.